addons/police/functions/fnc_createTeleport.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function creates teleport to all police stations except the one where the teleporter is located.
+ *
+ * Arguments:
+ * 0: Teleporter <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_flag] call afsk_police_fnc_createTeleport
+ *
+ * Public: No
+ */
+
+params ["_flag"];
+private _logic = _flag getVariable ["policeStation"];
```

Comment:
This variable is not published to clients by server.

Reply:
Good catch.